homeassistant/components/solvis_heating/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "domain": "solvis_heating",
+  "name": "Solvis Heating",
+  "codeowners": ["@lurchi70"],
+  "config_flow": true,
+  "dependencies": [],
+  "documentation": "https://www.home-assistant.io/integrations/solvis_heating",
+  "homekit": {},
+  "iot_class": "local_polling",
+  "requirements": ["requests==2.28.2"],
```

Comment:
I checked that, and was sure "requests" is a pypi package.
--> https://pypi.org/project/requests/

Reply:
It is a pypi package, but you should handle all of the requests calls in your own pypi package. 

As an aside, it is preferred for you to use aiohttp instead of request as it is async by nature.